proposals/dao/fip_60.ts

```diff
@@ -0,0 +1,255 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { deploy as deploySTW } from '@scripts/deploy/deployStakingTokenWrapper';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+chai.use(CBN(ethers.BigNumber));
+
+const UNDERLYINGS = [
```

Comment:
Please comment what these tokens are

Reply:
changed to use imported tokens to be more accurate